apps/stripe/src/app/api/saleor/with-recipient-verification.ts

```diff
@@ -0,0 +1,21 @@
+import { NextAppRouterSyncWebhookHandler } from "@saleor/app-sdk/handlers/next-app-router";
+import { WebhookContext } from "@saleor/app-sdk/handlers/shared";
+import { NextRequest } from "next/server";
+
+//Graphql part of the fragment responsible for recipient
+type PayloadPartial = { readonly recipient?: { id: string } | null };
```

Comment:
I think you can reuse `EventMetadataFragment['recipient']` from generated graphql